tofhir-log-server/src/main/scala/io/tofhir/log/server/ToFhirHttpServer.scala

```diff
@@ -1,11 +1,11 @@
-package io.tofhir.server
+package io.tofhir.log.server
```

Comment:
I suggest to rename ToFhirHttpServer also to sth like ToFhirLogHttpServer

Reply:
I intended to move this class to 'tofhir-server-common' since it's identical to the one in 'tofhir-server.'